go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/oasisprotocol/ledger-oasis-go
+module github.com/oasisprotocol/oasis-core-ledger
 
 go 1.13
```

Comment:
Should we bump this to 1.14 while we're here (oasis-core uses 1.14)?

Reply:
Agreed. Done.